lib/src/json_document_transformer.dart

```diff
@@ -7,7 +7,6 @@ import 'dart:convert';
 import 'package:async/async.dart';
 
 import '../stream_channel.dart';
```

Comment:
Can you switch these? Better to not import the 'root' library...

Reply:
Unfortunately / oddly, `class StreamChannel` is declared in `lib/stream_channel.dart`, and `lib/stream_channel.dart` exports the lib/src stuff. So we _must_ import the root library.

Reply:
Sad... 😞

Reply:
It is... odd. I think a non-breaking change to move the declaration to src/ and export from lib/stream_channel.dart.